### PR TITLE
show private repositories. Closes #15

### LINKED ISSUE #15: Repository list for logged in user doesn't show private repos
When clicking `Repositories` on the authenticated user profile screen, the list of repositories that shows is only public (unless actually searched for --> that call will show a private repo)

This is most likely due to endpoint that's hit by `getRepositories` in the `user.actions`. Although `/users/${user}/repos` is correct for other users --> we'll have to conditionally set the endpoint to be `/user/repos` for the authenticated user in order to see private repos in the list.

### PR DESCRIPTION
**Pull request by hosmelq**
Screenshot for repo list of the authenticated user.

screen shot 2017-07-12 at 9 08 24 pm

### CODE REVIEW

File: src/components/repository-list-item.component.js [hunk 1]
```diff
@@ -27,14 +27,17 @@ export const RepositoryListItem = ({ repository, navigation }: Props) => (
...
-      <Text style={styles.title}>
-        {repository.name}{repository.description && '\n'}
-      </Text>
-      <Text>
+    <View style={styles.repositoryContainer}>
+      <View style={styles.titleWrapper}>
+        <Text style={styles.title}>
+          {repository.name}
+        </Text>
+        {repository.private && <Text style={styles.private}>Private</Text>}
```
Comment (housseindjirdeh) [inline src/components/repository-list-item.component.js]: I love this :+1:

Comment (hosmelq) [inline src/components/repository-list-item.component.js]: I updated the description with a screenshot

**Comment by hosmelq** [comment]
I updated the description with a screenshot

**Comment by housseindjirdeh** [comment]
Beauty <3

**Pull request merged**